sim/dex.js

```diff
@@ -507,16 +507,36 @@ class ModdedDex {
 						ban = 'Rule:' + subformat.name;
 					} else {
 						ban = ban.toLowerCase();
-						let baseForme = false;
-						if (ban.endsWith('-base')) {
-							baseForme = true;
-							ban = ban.substr(0, ban.length - 5);
+						let gtIndex = ban.lastIndexOf('>');
+						let limit = 0;
+						if (gtIndex >= 0 && /^[0-9]+$/.test(ban.slice(gtIndex + 1).trim())) {
```

Comment:
We should probably not parse these things twice.

Especially not in `getFormat`... in hindsight, having a separate function to parse a banlist is probably a good idea.

Reply:
A separate function would be slightly less efficient for regular formats since custom formats need to validate each ban with <code>Dex.dataSearch</code>. Is that alright?

Reply:
I want to validate bans in general, yes. `dataSearch` is probably the wrong way to validate them, though.

Reply:
What would you suggest? I've used <code>dataSearch</code> for aliases but <code>getEffect</code> could work.

Reply:
I would actually individually use `getItem`, `getAbility`, etc